packages/asset-uploader/tsconfig.json

```diff
@@ -3,10 +3,13 @@
   "extends": "../../node_modules/@webstudio-is/tsconfig/base.json",
   "include": ["./src/**/*.ts"],
   "compilerOptions": {
+    "composite": true,
```

Comment:
can't this be in the packages/tsconfig?

Reply:
> the configs can probably be deduped better later - i didnt attempt to do this just yet

Reply:
you mean its not that easy to just put this property into the package? I am not sure what is this a problem to just move the flag to the central config

Reply:
ah already resolved